index.js

```diff
@@ -218,27 +218,23 @@ minimum value for how long a non-cached analyses will take
     }
   }
 
-  async getStatusOrIssues (uuid, url, inputApiUrl) {
+  async getStatusOrIssues (uuid, url) {
```

Comment:
Nice cleanup. (It had been in the back of my mind, but never had the stamina to follow through on this.)